website/prebuild.py

```diff
@@ -142,15 +142,36 @@ def label_root_file(
     add_frontmatter(f, f.read_text(), [title], top_nav_order, has_children)
 
 
+def unlink_dir(path: Path) -> None:
+    # TODO: Use path.walk once we require Python 3.12.
+    for root, dirs, files in os.walk(path, topdown=False):
+        root_path = Path(root)
+        for name in files:
+            (root_path / name).unlink()
+        for name in dirs:
+            (root_path / name).rmdir()
```

Comment:
Had you considered making use of `shutil.rmtree(path)`?

Reply:
I was unaware of its existence :) Switched to using it.